### PR TITLE
Enable WhatsNewStudy for 1.51 stable

### PR DESCRIPTION
**Pull request by simonhong**
Existing user will see whats-new tab once  when they're reached to 1.51.

### CODE REVIEW

**Comment by stephendonner** [comment]
## Verified `PASSED` using both `1.50.121` and `1.51.109`

### Steps:
1. installed `1.50.121`
2. launched Brave using `./Brave\ Browser --update-feed-url=https://updates.bravesoftware.com/sparkle/Brave-Browser/test/appcast.xml --variations-server-url=https://variations.bravesoftware.com/seed`
3. opened `brave://settings/help`
4. checked for updates
5. waited
6. clicked on `Relaunch`

### Confirmed I got the `What's New` page (`https://brave.com/whats-new`) in a foregrounded, focused tab

example | example
---------|---------
Screen Shot 2023-05-01 at 8 12 38 PM | Screen Shot 2023-05-01 at 8 13 00 PM

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` (staging) as per https://github.com/brave/brave-variations/pull/618#issuecomment-1530813375.

**Pull request merged**
